paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -124,6 +126,11 @@ internal fun PaymentSheetScreen(
     }
 }
 
+private val BaseSheetViewModel.contentVisible: StateFlow<Boolean>
+    get() {
+        return ((this as? PaymentSheetViewModel)?.contentVisible ?: stateFlowOf(true))
```

Comment:
Non-blocking but any way we could avoid avoid having to reference `PaymentSheetViewModel`?

Reply:
2 ways I can think of:
* Introduce an interface, and do the same as? check with it instead
* Wrap the screens in a way that knows the difference between PS/FC, and do it there

Reply:
I think the first solution makes the most sense. Rather have an abstraction that the view models can implement to make this more clear. Can be done in another PR.

Reply:
I actually leaned towards option 2. Let me know what you think.

I kinda like it because I think it makes our lives easier when things diverge in the future.